exchanges/retry/package.json

```diff
@@ -0,0 +1,58 @@
+{
+  "name": "@urql/exchange-retry",
+  "version": "0.1.0",
+  "description": "An exchange for operation retry support in urql",
+  "sideEffects": false,
+  "homepage": "https://formidable.com/open-source/urql/docs/",
+  "bugs": "https://github.com/FormidableLabs/urql/issues",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/FormidableLabs/urql.git",
+    "directory": "exchanges/retry"
+  },
+  "keywords": [
+    "urql",
+    "graphql client",
+    "formidablelabs",
+    "exchanges",
+    "react",
+    "retry"
+  ],
+  "main": "dist/urql-exchange-retry.cjs.js",
```

Comment:
Should we add an exports field similar to urql-react?

Reply:
That isn’t strictly speaking required unless we do that for all packages. It’s only required for `/extras` right now to make sure that Node does the right thing.

So we should probably file a ticket to update `prepare` and all packages